### ISSUE TITLE
SidebarView documentation: connectView should be setView

### ISSUE THREAD

**Issue by mantielero** [frescobaldi/qpageview#3]
In the [documentation](https://qpageview.org/sidebarview.html?highlight=sidebar#qpageview.sidebarview.SidebarView), it says:
> Use connectView() to connect a View, and it automatically shows the pages, also when the view is changed.

But I don't see `connectView` in the API. I though that it might refer to `setView`, but in that case I am not using it properlly. Could you post a simple example? (no need for a working example, just something that points me in the right direction)

What I am trying to do is to put a SidebarView and a View showing the page selected in the SidebarView. My code, right now looks like this:
```python
from PyQt5.QtWidgets import QMainWindow, QApplication, QMenu, \
    QAction, QFileDialog, qApp, QDockWidget#QHBoxLayout
from PyQt5.QtGui import QIcon
from PyQt5.QtCore import Qt
from pathlib import Path

import qpageview
from qpageview.sidebarview import SidebarView

import sys

class MainWindow(QMainWindow):
    def __init__(self, *args, **kwargs):
        super(MainWindow, self).__init__(*args, **kwargs)

        self.setWindowTitle("PDF Editor")

        # Menu
        menubar = self.menuBar()
        fileMenu = menubar.addMenu('File')

        openFile = QAction(QIcon('open.png'), 'Open', self)
        openFile.setShortcut('Ctrl+O')
        openFile.setStatusTip('Open new File')
        openFile.triggered.connect(self.showDialog)

        exitAction = QAction('&Exit', self)        
        exitAction.setShortcut('Ctrl+Q')
        exitAction.setStatusTip('Exit application')
        exitAction.triggered.connect(qApp.quit)        

        fileMenu.addAction(openFile)
        fileMenu.addAction(exitAction)
 
        # Side      
        self.v = SidebarView()
        self.v.loadPdf("../pdf/Frances_01.pdf")
        self.dockWidget = QDockWidget('Pages', self)
        self.dockWidget.setWidget(self.v)
        self.dockWidget.setFloating(False) 
        self.dockWidget.show()       
        self.addDockWidget(Qt.LeftDockWidgetArea, self.dockWidget)
        self.v.show()


        # Central
        self.cv = qpageview.View()
        self.v.setView(self.cv)   #<--------------- NOT WORKING AS EXPECTED
        self.cv.show()

        self.setCentralWidget(self.cv)

    def showDialog(self):

        home_dir = str(Path.home())
        fname = QFileDialog.getOpenFileName(self, 'Open file', home_dir)

        if fname[0]:
            self.v.loadPdf(fname[0]) 

def main():
    app = QApplication(sys.argv)
    window = MainWindow()
    window.resize(1000, 800)
    window.show() 
    
    sys.exit(app.exec_())


if __name__ == '__main__':
    main()
```

So basically is about the "NOT WORKING AS EXPECTED" line.

**Comment by mantielero** [comment]
Just for the record, the problem with my code is that I have to replace:
```python
self.v.loadPdf("../pdf/Frances_01.pdf")
```
with:
```python
self.cv.loadPdf("../pdf/Frances_01.pdf")
```

None the less, there is a typo. In the documentation mentions `connectView` while it should say `setView`.

**Comment by wbsoft** [comment]
Sorry for replying late ;-) Indeed the file should be loaded to the main view; the sidebarview will automagically update :-)

Thanks for pointing me to the error in the SidebarView docstring!
